frontend/src/components/Utils/time.js

```diff
@@ -88,9 +64,9 @@ export function timezonesForCountry(countryName) {
  */
 export function getDateBarebones(msTimestamp, timezone=null) {
   if (timezone === null) {
-    return moment.tz(parseFloat(msTimestamp), '').format('YYYY-MM-DD');
+    timezone = ''; // Use GMT. 
   }
-  return moment.tz(parseFloat(msTimestamp), timezone).format('YYYY-MM-DD');
+  return moment.tz(parseFloat(msTimestamp), timezone).format(moment.HTML5_FMT.DATE);
```

Comment:
If these are the same, why did you change it? I think it was more clear the other way. 

If it is different though, you need to update the documentation.

Reply:
Consistency. Plus, if this format changes for whatever reason, we don't have to manually go and update it.